tests/utils/test_archive_scientific_readers.py

```
@@ -216,6 +216,37 @@ def test_read_7z_file_not_installed(self, tmp_path: Path) -> None:
+        fake_archive.password_protected = False
+        fake_archive.__enter__ = lambda s: fake_archive
+        fake_archive.__exit__ = MagicMock(return_value=False)
+
+        mock_py7zr = MagicMock()
+        mock_py7zr.SevenZipFile.return_value = fake_archive
+
+        archive_path = tmp_path / "test.7z"
+        archive_path.write_bytes(b"dummy")
+
+        with (
+            patch.object(_arc, "PY7ZR_AVAILABLE", True),
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)